worldedit_brush/init.lua

```diff
@@ -1,14 +1,32 @@
+worldedit.brush = {
+	enabled = true,
+	modname = minetest.get_current_modname(),
+}
+
+local brush = worldedit.brush
+
+-- check compatibility
 if minetest.raycast == nil then
-	print('[MOD] worldedit_brush is not compatible with your game, so it won\'t be loaded!')
+	brush.enabled = false
```

Comment:
What I meant by "disable" is that people can disable the brush mod in the <b>menu</b>, not this.

Reply:
Just telling the user that they can do it (themselves) should be alright.

Reply:
You can disable individual mods in the main menu here:
![bildschirmfoto_2018-04-10_00-00-20](https://user-images.githubusercontent.com/1042418/38525374-55d89f78-3c52-11e8-8f3f-00fbb1573074.png)

Users should be instructed to do this by showing a message in the log/chat.

Reply:
That's true, I hope people seeing the error message are able to figure out how to remove the mod from a subgame.